composer.json

```diff
@@ -110,6 +110,9 @@
         "sort-packages": true
     },
     "extra": {
+        "branch-alias": {
```

Comment:
Please revert this. You can perform this aliasing in your own project when requiring guzzle.